.github/workflows/pr-smoke-test.yml

```
@@ -49,8 +51,50 @@ jobs:
           DEFAULT_TIMEOUT_OVERRIDE: 120000
         run: |
           pnpx wc-e2e test:e2e plugins/woocommerce/tests/e2e/specs/smoke-tests/update-woocommerce.js
+
+      - name: Post Smoke tests results comment on PR
+        if: always()
+        uses: actions/github-script@v5
+        env: 
+          TITLE: 'External Smoke Test Resutls:'
+          SMOKE_TEST_URL: ${{ secrets.SMOKE_TEST_URL }}
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const script = require( './package/woocommerce/packages/js/e2e-environment/bin/post-results-to-github-pr.js' )
```

```suggestion
            const script = require( '@woocommerce/e2e-environment/bin/post-results-to-github-pr.js' )
```

I didn't test this change but it should work.

---

@MrJnrman Sorry, I was wrong here. It looks like the script is running in the Github context vs the repo context. We'll need to restore your original code here.